app/assets/javascripts/tags.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Let's remove this file using `git rm app/assets/javascripts/tags.coffee`. This both removes the file from the directory, and tells git about it.